gbdx_auth/gbdx_auth.py

```diff
@@ -36,7 +39,19 @@ def save_token(token):
     t = jwt.decode(access_token, verify=False)
     token['expires_at'] = t['exp']
 
-    s = OAuth2Session(token=token, auto_refresh_url=auth_url, token_updater=save_token)
+    # GBDX no longer uses client_id and client_secret for authentication.
+    #   These values are dummy values which are still required due to legacy code in gameplan auth.
+    #   They can be shared without creating security concern. Here we default to shared values
+    #   across all gbdx users and accounts so users do not need to supply client_id and client_secret
+    #   to initiate a GBDX session from an existing id_token or refresh_token
+    #
+    s = OAuth2Session(
+            token=token,
+            auto_refresh_url=auth_url,
+            token_updater=save_token,
+            auto_refresh_kwargs={'client_id':os.environ.get("GBDX_CLIENT_ID", "@Ces!!u7Jd;eRfB8ww93YK1H2Wi2;EcrjCB6GlXp"),
+                                 'client_secret':os.environ.get("GBDX_CLIENT_SECRET", "P47VXn;AbYdqcM686fCiJ_O1.9SYIpVuM5MoeFp8m5tYWWqLU6PEn0?mT?jMaqug6JIN=B4tpHXLSbh_F5!lxwO05TS@IJ=.uMC9OhPlNyByUwsNt-Xo.ANIxxO;s6Zn")}
```

Comment:
could these "dummy" values just be something like "dummy_client_id" etc? we'd be able to quickly tell that they're just dummies then, but not a big deal.